java/memory/src/main/java/org/apache/arrow/memory/BaseAllocator.java

```diff
@@ -276,9 +316,7 @@ public ArrowBuf buffer(final int initialRequestSize, BufferManager manager) {
 
     // round to next largest power of two if we're within a chunk since that is how our allocator
```

Comment:
this comment is out of date?

Reply:
@emkornfield Thank you for your careful review. I have revised it accordingly.